### PR TITLE
fix(NcListItem): force three-dot-menu (+ minor follow-ups)

### LINKED ISSUE #12921: Move *Remove group and members* into an action menu (again)
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Feature request

**Which Nextcloud Version are you currently using:** the one for 30

**Is your feature request related to a problem? Please describe.**

Bildschirmfoto vom 2024-08-08 12-05-16

^ is scary. It feels too easy to click that button accidentally. Also, the red icon feels alarming like this.

**Describe the solution you'd like**

Force an action menu for (group) participants.

**Describe alternatives you've considered**

N/a

**Additional context**

N/a

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Follow-up with some vue-lib changes adjustments
* Fix #12921

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [ ] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/RightSidebar/RightSidebar.vue [hunk 1]
```diff
@@ -22,9 +22,9 @@
 			<NcCounterBubble v-if="unreadMessagesCounter > 0"
 				class="chat-button-unread-messages-counter"
 				:class="{ 'chat-button-unread-messages-counter--highlighted': hasUnreadMentions }"
-				:type="hasUnreadMentions ? 'highlighted' : 'outlined'">
-				{{ unreadMessagesCounter }}
-			</NcCounterBubble>
+				:type="hasUnreadMentions ? 'highlighted' : 'outlined'"
+				:count="unreadMessagesCounter" />
+			<!-- FIXME looks weird -->
```
Comment (ShGKme) [inline src/components/RightSidebar/RightSidebar.vue:27]: ?

Comment (Antreesy) [reply src/components/RightSidebar/RightSidebar.vue:27]: whoops

Comment (Antreesy) [reply src/components/RightSidebar/RightSidebar.vue:27]: As per discussion with design team, will do that

**Review by DorraJaouad — APPROVED** [review]
🦅

**Pull request merged**